src/core/AndroidConvertor.js

```diff
@@ -0,0 +1,108 @@
+import BaseConvertor from './BaseConvertor'
+class AndroidConvertor extends BaseConvertor {
+
+    translateFromConversionItems(args) {
+        var availableLang = this._getAvailableLang(args);

```

Comment:
```suggestion
        let availableLang = this._getAvailableLang(args);
```
Je te recommande d'utiliser des let au lieu de var, et quand tu peux des const!